deploy/roles/postgresql-server/defaults/main.yml

```diff
@@ -211,7 +214,7 @@ pp_wal_sender_timeout: 60s
 pp_max_replication_slots: 0
 
 # master
-pp_syncronous_standby_names: ''
+pp_synchronous_standby_names: ''
```

Comment:
Nice catch :)  We just need to double check that the misspelled version isn't set in `aurora-internal` somewhere.